### DISCUSSION TITLE
ネザー 禍壺の砦E 開発状況

### DISCUSSION THREAD

**Discussion by anminmakura** [TUSB/TheUnusualSkyBlock#74] [エリア・建築の完成度・開発状況]
**壺内部の砦と見えている岩盤にについて**
座標地点 ネザー 1358 98 723
- [x] 土台の製作
- [x] 建築に着手
- [x] 装飾に着手
- 要望
某クッパ城をイメージして砦を作ってほしい。
ついでに中に入ったら見える岩盤をちょっと見えないように加工もしてほしい。

**Comment by anminmakura** [comment]
![1358_98_723](https://user-images.githubusercontent.com/39670701/96728892-84bca880-13ef-11eb-8bb9-0cdc2802e67d.png)

**Comment by anminmakura** [comment]
![2021-06-05_22 53 25](https://user-images.githubusercontent.com/39670701/120893975-e7964380-c650-11eb-83aa-1166afcc0a7d.png)
作りかけ。
誰かが着手しようとした痕跡あり、放置状態なので誰が制作陣募集。

**Comment by huyuyori** [comment]
![2021-06-07_08 42 11](https://user-images.githubusercontent.com/73624149/120943836-5b793e80-c76c-11eb-9305-7f87961b3d6b.png)
建築中
